Installing-Oppia-(Linux;-Python-3).md

```diff
@@ -182,7 +192,7 @@ exec "$SHELL"
    > [!WARNING]
```

Comment:
In "7." above -- let's change to "Add a new file called ..." (I know this was already there, but thought I'd mention it since we're updating this page fully)

Also this WARNING doesn't seem to be displaying correctly, ditto below/elsewhere. It's possible that those indicators don't work correctly in indented lists.